packages/devtools_app/test/flutter/story_of_layout/flex_test.dart

```diff
@@ -262,4 +263,38 @@ void main() {
       );
     }, skip: kIsWeb || !isLinux);
   });
+
+  test('sum', () {
+    expect(sum([1.0, 2.0, 3.0]), 6.0);
+  });
+
+  group('computeRenderSizes', () {
```

Comment:
What happens in the case where the items come out larger than the maxSize?

Test that we can switch to the scrolling mode if need be.